sql/parser/sql.y

```diff
@@ -1003,19 +1003,17 @@ set_reset_clause:
   SET set_rest {}
 
 show_stmt:
-  SHOW var_name
-  {
-    // It would be cleaner if we could have "SHOW DATABASES" and "SHOW
-    // TABLES" rules, but unfortunately DATABASES and TABLES are
-    // unreserved keywords and thus valid variable names and such
-    // rules would cause reduce/reduce conflicts.
-    if strings.EqualFold($2.String(), `"DATABASES"`) {
-      $$ = &ShowDatabases{}
-    } else if strings.EqualFold($2.String(), `"TABLES"`) {
-      $$ = &ShowTables{}
-    } else {
-      $$ = nil
```

Comment:
We allow `SET <var_name> = foo`. We should allow `SHOW <var_name>`. Right now, `DATABASE` is the only variable we support, though perhaps that should be `SESSION_DATABASE`.